README.md

```diff
@@ -13,7 +13,7 @@ For more information, see:
 
 ## Installation
 
-First install [Go 1.9.1](https://golang.org/doc/install) and [Glide](https://github.com/Masterminds/glide#install). Note that Glide requires an explicit GOPATH to be set.
+First install [Go 1.9.1](https://golang.org/doc/install) or later and [dep](https://golang.github.io/dep/docs/installation.html). Note that dep requires an explicit GOPATH to be set.
```

Comment:
Git won't let me make a comment down there, but... about 10 lines below this one, there's a `glide install` command in the "download and prepare" section of the docs.